src/main/kotlin/screeps/api/PathFinder.kt

```diff
@@ -32,12 +32,12 @@ external object PathFinder {
 }
 
 external interface SearchOptions : Options {
-    var roomCallback: ((String) -> PathFinder.CostMatrix)?
-    var plainCost: Int?
-    var swampCost: Int?
-    var flee: Boolean?
-    var maxOps: Int?
-    var maxRooms: Int?
-    var maxCost: Int?
-    var heuristicWeight: Double?
+    var roomCallback: ((String) -> PathFinder.CostMatrix)? get() = definedExternally; set(value) = definedExternally
+    var plainCost: Int? get() = definedExternally; set(value) = definedExternally
```

Comment:
all of those could be simplified like this `var plainCost: Int? = definedexternally`

Reply:
I'm getting "Property initializers are not allowed in interfaces" then

Reply:
My bad